src/components/inputsList/InputsList.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import Input from '../input/Input';
+
+const InputList = ({ inputs }) => (
```

Comment:
I think you could do this component id different way.
You don't need conditionally render some parts of table (like headers) depending on inputs.
The table is always tries to render everything.
```javascript
const InputList = ({ inputs }) => (
  <table>
    <thead>
      <tr>
      {
          headers.map( (header, index) => (
              <th key={index}>{header}</th>
          ));
      }
      </tr>
    </thead>
    <tbody>
    {
        inputs.map(input => (
            <Input key={input.id} {...input}  />
        ))
    }
    </tbody>

  </table>
);
```

Also you need to create some component which holds empty table logic (see an example of empty https://ibb.co/ccW7ev)

than the client (the code which uses this list) should do something like this:

```javascript
{
    hasInputs()
        ? <InputList inputs={inputs}>
        : <Empty /> // please not that the name is an example
}
```
This would allow to eliminate some ifs